frontend/package.json

```diff
@@ -6,26 +6,26 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
-    "lint": "vue-cli-service lint && npm run license-file-headers-add",
```

Comment:
where is this now?

Reply:
The license-file-headers-add script is called via lint-staged, so in .lintstagedrc.json.
It should not be included in the build, as it could worst case create pending file changes, which would probably impact build pipelines.
Headers should be created during development, before committing.

Reply:
makes sense. the backend build fails in this case is it possible to have the same behavior here?

Reply:
Yes, the license-check-and-add package offers a "check" option we could use in the build script (before starting the actual "vue-cli-service build" task). If it fails, the build fails. I can add it in a new commit

Reply:
Can you have a look to the build.yml there we have:

```yml
      # Test Phase

      - name: Test with npm
        ## TODO are there any frontend tests?
        run: npm --prefix frontend run license-check
````
I think it makes sense to add something similar

Reply:
Yes, we can add one here too. The build should fail as early as possible in this case (missing license headers) as it is not resource intensive. Where do you think we should add it in the build.yml file?

Reply:
As another step in the test phase

Reply:
Added the step in new commit